src/runs/execution/factory.ts

```diff
@@ -100,7 +100,8 @@ export function createChatLLM(run: Loaded<Run>, backend: LLMBackend = LLM_BACKEN
           },
           stopping: {
             ...parameters.stopping,
-            max_new_tokens: MAX_NEW_TOKENS
+            // TODO: temporary override, should be increased in vllm deployment
+            max_new_tokens: run.model.includes('granite') ? 1024 : MAX_NEW_TOKENS
```

Comment:
agent with max tokens 1024 is unusable really, should be increased before merging this